### ISSUE TITLE
Server Running on port 3000 does not serve any response to browser

### ISSUE THREAD

**Issue by kohakade** [YouHusam/hanx.js#9]
Hi,
I am new to hapijs and hnax.js boilerplate. I have configured the hanx.js on my system (Fedora 21). I ran development server and its running without any error on localhost:3000 but when i try to hit the same in browser of  same machine i get "This webpage is not available ERR_CONNECTION_REFUSED" and server not get request. But same code run on ubuntu machine its working properly.
     Deploying getting started Hapijs  server code works properly.

**Comment by ihorskyi** [comment]
@kohakade could you please post complete logs here from app startup & localhost:3000 hit?

**Comment by kohakade** [comment]
@ihorskyi thanks for response.
Here is running app startup with "good"  package is registered for console log.

![screenshot from 2016-02-15 11 57 57](https://cloud.githubusercontent.com/assets/6702584/13041242/7f4f221e-d3db-11e5-9cad-824c99c4307e.png)

Here is the screen after localhost:3000 hit

![screenshot from 2016-02-15 11 58 14](https://cloud.githubusercontent.com/assets/6702584/13041243/7f4fcbc4-d3db-11e5-956d-8ad7a0102c95.png)

**Comment by YouHusam** [comment]
have you tried http://127.0.0.1:3000 ?

**Comment by kohakade** [comment]
Yes i tried using loop back address and using ip address but got same result.
